src/pages/list/search/articles/index.tsx

```diff
@@ -0,0 +1,267 @@
+import { Icon as LegacyIcon, Form } from '@ant-design/compatible';
+import { Button, Card, Col, List, Row, Select, Tag } from 'antd';
+import React, { Component } from 'react';
+
+import { Dispatch } from 'redux';
+import { FormComponentProps } from 'antd/es/form';
+import { connect } from 'dva';
+import ArticleListContent from './components/ArticleListContent';
+import { StateType } from './model';
+import { ListItemDataType } from './data.d';
+import StandardFormRow from './components/StandardFormRow';
+import TagSelect from './components/TagSelect';
+import styles from './style.less';
+
+const { Option } = Select;
+const FormItem = Form.Item;
+
+const pageSize = 5;
+
+interface ArticlesProps extends FormComponentProps {
+  dispatch: Dispatch<any>;
+  listAndsearchAndarticles: StateType;
+  loading: boolean;
+}
+
+class Articles extends Component<ArticlesProps> {
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch({
+      type: 'listAndsearchAndarticles/fetch',
+      payload: {
+        count: 5,
+      },
+    });
+  }
+
+  setOwner = () => {
+    const { form } = this.props;
+    form.setFieldsValue({
+      owner: ['wzj'],
+    });
+  };
+
+  fetchMore = () => {
+    const { dispatch } = this.props;
+    dispatch({
+      type: 'listAndsearchAndarticles/appendFetch',
+      payload: {
+        count: pageSize,
+      },
+    });
+  };
+
+  render() {
+    const {
+      form,
+      listAndsearchAndarticles: { list },
+      loading,
+    } = this.props;
+    const { getFieldDecorator } = form;
+
+    const owners = [
+      {
+        id: 'wzj',
+        name: '我自己',
+      },
+      {
+        id: 'wjh',
+        name: '吴家豪',
+      },
+      {
+        id: 'zxx',
+        name: '周星星',
+      },
+      {
+        id: 'zly',
+        name: '赵丽颖',
+      },
+      {
+        id: 'ym',
+        name: '姚明',
+      },
+    ];
+
+    const IconText: React.FC<{
+      type: string;
+      text: React.ReactNode;
+    }> = ({ type, text }) => (
+      <span>
+        <LegacyIcon type={type} style={{ marginRight: 8 }} />
+        {text}
+      </span>
+    );
+
+    const formItemLayout = {
+      wrapperCol: {
+        xs: { span: 24 },
+        sm: { span: 24 },
+        md: { span: 12 },
+      },
+    };
+
+    const loadMore = list.length > 0 && (
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 0](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)